src/round.rs

```diff
@@ -220,6 +221,12 @@ pub struct Round<Id: Hash + Eq, H: Hash + Eq, N, Signature> {
 	completable: bool, // whether the round is completable
 }
 
+pub struct ImportResult<Id, P, Signature> {
+	pub valid_voter: bool,
+	pub duplicated: bool,
+	pub equivocation: Option<Equivocation<Id, P, Signature>>,
```

Comment:
docs on public items - also, pub fields on public items is a bit dangerous for backwards compatibility.

Reply:
do they need to be `pub` or just `pub(crate)`?